types/node/v15/child_process.d.ts

```diff
@@ -473,7 +473,7 @@ declare module 'child_process' {
     }
     interface SpawnSyncReturns<T> {
         pid: number;
-        output: string[];
+        output: Array<T | null>;
```

Comment:
You can remove the v15 changes, I intent to remove the typings for v15 as it is not supported: https://nodejs.org/en/about/releases/